### PR TITLE
Capabilities and limitation to Public Cloud Databases

### PR DESCRIPTION
**Pull request by Pirionfr**
add Capabilities and limitation to Public Cloud Databases

### CODE REVIEW

File: pages/platform/databases/information_05_capabilities/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: Capabilities and limitation
+slug: capabilities
+excerpt: Discover the capabilities and limitations of Public Cloud Databases
+section: General information
+order: 040
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.en-gb.md]: Bump to 050 as 040 is for information_04_faq

File: pages/platform/databases/information_05_capabilities/guide.pt-pt.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+Choose from a large portfolio of popular engines.
+
+We continuously improve our offers. You can follow and submit ideas to add to our roadmap at https://github.com/ovh/public-cloud-roadmap/projects/2.
+
+## Database Engines
+
+We currently offer seven database engines:
+
+### [MySQL](/publiccloud/databases/mysql/capabilities/)
+MySQL is an open-source fully-managed relational database service. It's fast, reliable, scalable, and user-friendly
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.pt-pt.md]: Missing the ending "."

File: pages/platform/databases/information_05_capabilities/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+Choose from a large portfolio of popular engines.
+
+We continuously improve our offers. You can follow and submit ideas to add to our roadmap at https://github.com/ovh/public-cloud-roadmap/projects/2.
+
+## Database Engines
+
+We currently offer seven database engines:
+
+### [MySQL](/publiccloud/databases/mysql/capabilities/)
+MySQL is an open-source fully-managed relational database service. It's fast, reliable, scalable, and user-friendly
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.en-gb.md]: Missing the ending "."

File: pages/platform/databases/information_05_capabilities/guide.en-gb.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+### [PostgreSQL](/publiccloud/databases/postgresql/capabilities/)
+PostgreSQL is an open-source fully managed and hosted relational database service. It's a high-performance engine that offers flexibility through variety of included advanced extensions
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.en-gb.md]: Missing the ending "."

File: pages/platform/databases/information_05_capabilities/guide.en-gb.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+We currently offer seven database engines:
+
+### [MySQL](/publiccloud/databases/mysql/capabilities/)
+MySQL is an open-source fully-managed relational database service. It's fast, reliable, scalable, and user-friendly
+
+### [PostgreSQL](/publiccloud/databases/postgresql/capabilities/)
+PostgreSQL is an open-source fully managed and hosted relational database service. It's a high-performance engine that offers flexibility through variety of included advanced extensions 
+
+### [MongoDB](/publiccloud/databases/mongodb/capabilities/)
+MongoDB is fully managed document-oriented database management system scalable and does not require a predefined data schema
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.en-gb.md]: Missing the ending "."

File: pages/platform/databases/information_05_capabilities/guide.en-gb.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+Three plans are available:
+
+### Essential
+The Essential plan typically offers the basic features and functionalities required for managing a database. 
+It is designed for tests, development environments and non-critical projects.
+
+### Business
+The Business plan is more comprehensive and suitable for organizations with higher database demands.
+This is the first production-ready offering
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.en-gb.md]: Missing the ending "."

File: pages/platform/databases/information_05_capabilities/guide.en-gb.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+(1) depend on the plan
+
+# Specific feature Plans
+
+Three plans are available:
+
+### Essential
+The Essential plan typically offers the basic features and functionalities required for managing a database. 
+It is designed for tests, development environments and non-critical projects.
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.en-gb.md]: Suggestion: "It is designed for test and development environments or non-critical projects."

File: pages/platform/databases/information_05_capabilities/guide.en-gb.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+| Read-only nodes                                                                                              | ✓     | ✓          | ✓                |       |      |            |           |
+| Point in Time Recovery                                                                                       | ✓     | ✓          | ✓ <sup>(1)</sup> |       |      |            |           |
+| Database forking                                                                                             | ✓     | ✓          | ✓                | ✓     | ✓    | ✓          | ✓         |
+| Terraform Provider support ([more information](https://registry.terraform.io/providers/ovh/ovh/latest/docs)) | ✓     | ✓          | ✓                | ✓     | ✓    | ✓          | ✓         |
+| Private network connectivity (vRack)                                                                         | ✓     | ✓          | ✓                | ✓     | ✓    | ✓          | ✓         |
+
+(1) depend on the plan
+
+# Specific feature Plans
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.en-gb.md]: Should be "##" level.

File: pages/platform/databases/information_05_capabilities/guide.en-gb.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+It is designed for tests, development environments and non-critical projects.
+
+### Business
+The Business plan is more comprehensive and suitable for organizations with higher database demands.
+This is the first production-ready offering
+
+### Enterprise
+The enterprise pricing plan is the most advanced and robust option available for managed databases. 
+It is dedicated to organizations with complex and high-performance database requirements.
+
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.en-gb.md:86]: Maybe create a dedicated section like "### Plan capabilities overview" .

File: pages/platform/databases/information_05_capabilities/guide.en-gb.md [hunk 10]
```diff
@@ -0,0 +1,105 @@
+---
+title: Capabilities and limitation
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.en-gb.md]: "Limitations" with a capital "L" and a trailing "s".

File: pages/platform/databases/information_05_capabilities/guide.en-gb.md [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+| Feature           | Essential | Business | Enterprise |
+|-------------------|-----------|----------|------------|
+| SLA               | None      | 99.9%    | 99.95%     |
+| High availability |           | ✓        | ✓          |
+
+
+Your choice of plan affects the number of nodes your cluster can run.
+
+
+## Last Updates
```
Comment (thild42) [inline pages/platform/databases/information_05_capabilities/guide.en-gb.md:99]: Add lastest update for secondary region.

**Review by thild42 — CHANGES_REQUESTED** [review]

**Review by thild42 — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @Pirionfr

**Pull request merged**